### PR TITLE
Add profile for the reMarkable2 ePaper tablet

### PR DESCRIPTION
**Pull request by Locutus-code**
Tiny PR for the screen properties of the reMarkable tablet.

I've been happy so far using this to convert files for the tablet, and a default profile would be nice :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -449,6 +449,7 @@ Config:
     	    - KoF     ( 1440x1920 ) - Kobo Forma
     	    - KoS     ( 1440x1920 ) - Kobo Sage
     	    - KoE     ( 1404x1872 ) - Kobo Elipsa
+			- RM      ( 1404x1872 ) - reMarkable 2
```
Comment (celogeek) [inline README.md]: Can you fix the spaces here.

File: internal/converter/profiles/converter_profiles.go [hunk 2]
```diff
@@ -49,6 +49,7 @@ func New() Profiles {
 		{"KoF", "Kobo Forma", 1440, 1920},
 		{"KoS", "Kobo Sage", 1440, 1920},
 		{"KoE", "Kobo Elipsa", 1404, 1872},
+		{"RM2", "reMarkable 2", 1404, 1872},
```
Comment (celogeek) [inline internal/converter/profiles/converter_profiles.go:53]: reMarkable is a Kobo device ?
if not, can you add a comment like
```
// Other
```
or 
```
// reMarkable
```

**Comment by celogeek** [comment]
I also see this device seems to have the same dimension as the Kobo Elipsa. I guess you where using that one up to now ?
So except to have a better comment on the profile you are using, the result will be exactly the same.

**Review by celogeek — CHANGES_REQUESTED** [review]
Thanks for the PR.
I left some comments.

File: README.md (hunk 1 above)
Comment (Locutus-code) [reply README.md]: Done

File: internal/converter/profiles/converter_profiles.go (hunk 2 above)
Comment (Locutus-code) [reply internal/converter/profiles/converter_profiles.go:53]: Added `// reMarkable`

There's also a reMarkable 1, but i think they are quite rare and a version 3 is supposed to come out next year. So we could still add those later.

**Comment by Locutus-code** [comment]
Fixed review comments, and just to confirm. Yes, the point was just to have an explicit profile for the remarkable2.

The RM2 is really picky with handling comics PDF's/ePub's, and this tool does actually a really good job at pre-processing the files easilly into a format the RM2 handles well.

Having a specific profile would be nice for people searching for a convertor.

File: README.md [hunk 3]
```diff
@@ -449,6 +449,7 @@ Config:
     	    - KoF     ( 1440x1920 ) - Kobo Forma
     	    - KoS     ( 1440x1920 ) - Kobo Sage
     	    - KoE     ( 1404x1872 ) - Kobo Elipsa
+    	    - RM      ( 1404x1872 ) - reMarkable 2
```
Comment (vincent-ogury) [inline README.md:452]: can you set RM2 here ?

usually I do
```
go run . -help
```
then I copy paste it in the readme
and to ensure I don't put my default, I do a 
```
go run . -reset
```

But you may want to save your config so you can do something like
```
mv ~/.go-comic-converter.yaml ~/.go-comic-converter.yaml.old
go run . -help
mv ~/.go-comic-converter.yaml.old ~/.go-comic-converter.yaml
```

Comment (celogeek) [inline README.md:452]: can you set RM2 here ?

usually I do
```
go run . -help
```
then I copy paste it in the readme
and to ensure I don't put my default, I do a 
```
go run . -reset
```

But you may want to save your config so you can do something like
```
mv ~/.go-comic-converter.yaml ~/.go-comic-converter.yaml.old
go run . -help
mv ~/.go-comic-converter.yaml.old ~/.go-comic-converter.yaml
```

File: internal/converter/profiles/converter_profiles.go (hunk 2 above)
Comment (celogeek) [reply internal/converter/profiles/converter_profiles.go:53]: ok

**Comment by celogeek** [comment]
I will merge and fix it afterward

**Pull request merged**